qa/restart-upgrade/src/test/resources/processor/UploadModelRequestBody.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Are configurations duplicated across restart and rolling? If so, can we avoid duplicating?

Reply:
So, the files are kept at 2 places because the the test folder only picks resources if it is present under the same path. I have tried to keep it common but it does not work. 

For example
```
qa:
      restart-upgrade:
                     test
                            code
                            resources
       rolling-upgrade:
                     test
                            code
                            resources
```

I have tried all combinations like 

```
qa:
      restart-upgrade:
                     test
                            code
       rolling-upgrade:
                     test
                            code
       src
            test
                 resources
```

```
qa:
      resources:
      restart-upgrade:
                     test
                            code
       rolling-upgrade:
                     test
                            code
```

But it didn't work. 

Even I tried to change the path from gradle from where it picks the resources but it didn't work because after ./gradlew build it creates a file location in resources and from there it picks the resource. So i cannot change it

Reply:
ah I see makes sense.